src/mbgl/gl/layer_group_gl.cpp

```diff
@@ -28,8 +28,10 @@ void TileLayerGroupGL::upload(gfx::UploadPass& uploadPass) {
         return;
     }
 
-    MLN_TRACE_FUNC()
-    MLN_ZONE_STR(name)
+    {
+        MLN_TRACE_FUNC()
+        MLN_ZONE_STR(name)
+    }
```

Comment:
What about this? Seems to be duplicated?

Reply:
`MLN_ZONE_STR` adds a string text to the Zone. From Tracy manual:
```
Use the ZoneText(text, size) macro to add a custom text string that the profiler will display along
with the zone information (for example, name of the file you are opening).
```